Task 8/script.js

```diff
@@ -5,3 +5,36 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+
+function Calculator() {}
```

Comment:
Bendrai žiūrint, manau tai pats logiškiausias sprendimas, jog visus metodus prototipuojate, užuot viską jungus kiekvienam naujam instance per "this" keywordą.

Bravo.

1/1